modules/chapter3/pages/section2.adoc

```diff
@@ -1,3 +1,150 @@
+
+2. Open the JupyterLab interface of the notebook.
+
+3. Press kbd:[Ctrl+Shift+L] to open a new launcher.
+
+4. In the btn:[Notebook] section, click btn:[Python 3.9].
++
+image::jupyter-notebook-create.png[]
++
+JuypterLab creates a new `Untitled.ipynb` notebook file.
+
+5. Press kbd:[Ctrl+S] to save the notebook.
+Enter a name for the notebook and click btn:[Rename].
++
+image::jupyter-notebook-rename.png[]
++
+You have now created your first notebook.
++
+[IMPORTANT]
+====
+Before creating a notebook, verify that your are at the directory where you wish to create the notebook file.
+Use the JupyterLab file browser to navigate to the right directory.
+====
+
+
+== Exercise: Importing a Notebook from Git
+
+1. In the same JupyterLab window, click the Git icon.
+
+2. Click btn:[Clone a repository].
++
+image::git-clone-menu.png[width=40%,align="center"]
+
+3. Enter https://github.com/RedHatQuickCourses/rhods-qc-apps as the repository, and click btn:[Clone].
+
+4. Navigate to any of the subdirectories of `rhods-qc-apps` and double click a notebook file to open it.
+
+
+== Exercise: Creating a Notebook in a Git Repository
+
+1. Create a public GitHub repository with your GitHub account.
+Refer to https://docs.github.com/en/get-started/quickstart/create-a-repo if you need help with this step.
+
+2. Return to the JupyterLab window that you have used in the previous exercise.
+Click the Git icon.
+
+3. Click btn:[Clone a repository].
++
+image::git-clone-menu.png[width=40%,align="center"]
+
+4. Enter the URL of your Git repository and click btn:[Clone].
++
+image::git-clone-modal.png[align="center"]
+
+5. In the file browser, navigate to the repository directory.
+
+6. In this directory, create a notebook file called `hello.ipynb`.
+You can follow the steps given in the first exercise of this page.
+
+7. Add the following code in the first cell:
++
+[source,python]
+----
+print("Hello world")
+----
+
+8. Press kbd:[Ctrl+Enter] to execute the cell.
+Your notebook should display as follows:
++
+image::hello-world-notebook.png[align="center"]
+
+9. Press btn:[Ctrl+S] to save the notebook.
+Alternatively, you can click menu:File[Save Notebook].
+
+10. Click the Git tab in the side bar.
+Git displays the newly added file:
++
+image::git-untracked-file.png[]
+
+11. Stage the `hello.ipynb` file.
+Right click the file and click btn:[Track].
++
+image::git-track-file.png[]
++
+The file is now staged.
++
+image::git-file-staged.png[]
+
+12. Enter the commit message and the description at the bottom, and click btn:[Commit].
++
+image::git-commit-changes.png[]
++
+At this point, JupyterLab prompts for the committer information.
+Enter your name and email.
+
+13. Push your changes to GitHub.
+Click the btn:[up arrow cloud] icon at the top.
++
+image::git-push-icon.png[]
++
+At this point, JupyterLab requires you to authenticate into GitHub.
+Enter your GitHub username and token.
```

Comment:
Any way to avoid it? I think we don't show people how to create a PAT Github token? Maybe we need to do this in the very first exercise involving git auth with screenshots. Or figure out a way to store the creds in a secret?

Reply:
No easy way to do this from a data scientist perspective.

The "easiest" way is to tweak the workbench CRs, but that's out of the scope of what a data scientist should do imo.

Another more advanced option is configuring webhooks in kubernetes, whic in my opinion is a 
topic for admins https://chat.google.com/room/AAAAWmAOoJE/g0jUX-Vmb40

Reply:
I have added a note listing some workarounds for this, but i have not provided too much detail. The Secret option is imo more on the admin side.